src/TestHelper/src/System/Diagnostics/RemoteExecutorTestBase.nouap.cs

```diff
@@ -0,0 +1,66 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
I was trying to name the files with the targetGroup id's. but I don't have any objection to rename it as you suggested.